src/Field/Field.php

```diff
@@ -39,7 +39,21 @@ public function getArguments() : \Graphpinator\Argument\ArgumentSet
 
     public function printSchema() : string
     {
-        return $this->getName() . $this->printArguments() . ': ' . $this->getType()->printName() . $this->printDeprecated();
+        return $this->printDescription() . $this->getName() . $this->printArguments() . ': ' . $this->getType()->printName() . $this->printDeprecated();
+    }
+
+    public function printDescription() : string
```

Comment:
private + this function might get reused - consider moving it into TOptionalDescription